templates/web/base/admin/templates_index.html

```diff
@@ -0,0 +1,11 @@
+[% INCLUDE 'admin/header.html' title=loc('Response Templates') -%]
+
+<ul>
+    [% FOR body IN bodies %]
+        <li>
+            <a href="/admin/templates/[% body.id %]/[% t.id %]">[% body.name %]</a>
```

Comment:
Oops. I'd intended this to just be a list of links to bodies, have removed the `/[% t.id %]`.